src/main/resources/db/migration/v0.37.0.64__curated_variant_genomic_location_association.sql

```diff
@@ -0,0 +1,180 @@
+ALTER TABLE ONLY allelevariantassociation ADD CONSTRAINT allelevariantassociation_updatedby_id_fk FOREIGN KEY (updatedby_id) REFERENCES person(id);
+ALTER TABLE ONLY allelevariantassociation ADD CONSTRAINT allelevariantassociation_evidencecode_id_fk FOREIGN KEY (evidencecode_id) REFERENCES ontologyterm(id);
+ALTER TABLE ONLY allelevariantassociation ADD CONSTRAINT allelevariantassociation_relatednote_id_fk FOREIGN KEY (relatednote_id) REFERENCES note(id);
+ALTER TABLE ONLY allelevariantassociation ADD CONSTRAINT allelevariantassociation_relation_id_fk FOREIGN KEY (relation_id) REFERENCES vocabularyterm(id);
+ALTER TABLE ONLY allelevariantassociation ADD CONSTRAINT allelevariantassociation_aasubject_id_fk FOREIGN KEY (alleleassociationsubject_id) REFERENCES allele(id);
+ALTER TABLE ONLY allelevariantassociation ADD CONSTRAINT allelevariantassociation_avaobject_id_fk FOREIGN KEY (allelevariantassociationobject_id) REFERENCES variant(id);
+
+ALTER TABLE ONLY allelevariantassociation_informationcontententity ADD CONSTRAINT allelevariantassociation_ice_association_id_fk FOREIGN KEY (association_id) REFERENCES allelevariantassociation(id);
+ALTER TABLE ONLY allelevariantassociation_informationcontententity ADD CONSTRAINT allelevariantassociation_ice_evidence_id_fk FOREIGN KEY (evidence_id) REFERENCES informationcontententity(id);
+
+INSERT INTO vocabularytermset (id, name, vocabularylabel, vocabularytermsetvocabulary_id)
+	SELECT nextval('vocabularytermset_seq'), 'Allele Variant Association Relation', 'allele_variant_relation', id FROM vocabulary WHERE vocabularylabel = 'allele_relation';
+
+INSERT INTO vocabularyterm (id, name, vocabulary_id) SELECT nextval('vocabularyterm_seq'), 'has_variant', id FROM vocabulary WHERE vocabularylabel = 'allele_relation';
+
+INSERT INTO vocabularytermset_vocabularyterm (vocabularytermsets_id, memberterms_id) 
+	WITH
+  	t1 AS (
+    	SELECT id FROM vocabularytermset WHERE vocabularylabel = 'allele_variant_relation'
+  	),
+  	t2 AS (
+    	SELECT id FROM vocabularyterm WHERE name = 'has_variant' AND vocabulary_id = (
+    		SELECT id FROM vocabulary WHERE vocabularylabel = 'allele_relation'
+    	)
+  	)
+  	SELECT t1.id, t2.id FROM t1,t2;
+
+DELETE FROM bulkloadfileexception WHERE bulkloadfilehistory_id IN (SELECT id FROM bulkloadfilehistory WHERE bulkload_id IN (SELECT id FROM bulkload WHERE backendbulkloadtype = 'VARIANT'));
+DELETE FROM bulkloadfilehistory WHERE bulkload_id IN (SELECT id FROM bulkload WHERE backendbulkloadtype = 'VARIANT');
+DELETE FROM bulkloadfile WHERE id NOT IN (SELECT DISTINCT bulkloadfile_id FROM bulkloadfilehistory);
+DELETE FROM bulkmanualload WHERE id IN (SELECT id FROM bulkload WHERE backendbulkloadtype = 'VARIANT');
+DELETE FROM bulkload WHERE backendbulkloadtype = 'VARIANT';
+DELETE FROM bulkloadgroup WHERE name = 'Direct (LinkML) DQM Variant Loads';
+
+INSERT INTO bulkloadgroup (id, name) VALUES (nextval('bulkloadgroup_seq'), 'Variant Bulk Loads');
```

Comment:
Can we change the name of this to: `File Management System (FMS) Variant Loads`
This way all the FMS groups will clump together on the data loads page.

Reply:
Sure, will rename the other FMS loads too

Reply:
Done